### PR TITLE
Feature/73 travis database implementation

### PR DESCRIPTION
**Pull request by sjongejan**

### CODE REVIEW

File: .env.travis [hunk 1]
```diff
@@ -0,0 +1,14 @@
+APP_ENV=testing
+APP_DEBUG=true
+APP_KEY=Tr@v1S_K3y
+APP_TIMEZONE=UTC
+
+DB_CONNECTION=mysql
+DB_HOST=127.0.0.1
+DB_PORT=3306
+DB_DATABASE=header-checker-rest
```
Comment (FaaPz) [inline .env.travis:9]: `xebia_header_checker` 😉

File: .env.travis [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+DB_USERNAME=root
+DB_PASSWORD=
```
Comment (FaaPz) [inline .env.travis:11]: Should this be blank?

File: tests/TestCase.php [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 abstract class TestCase extends Laravel\Lumen\Testing\TestCase
 {
+    use \Laravel\Lumen\Testing\DatabaseMigrations;
```
Comment (FaaPz) [inline tests/TestCase.php]: Please import this.

File: app/Parsers/RequestHeaderParser/RequestHeaderParser.php [hunk 4]
```diff
@@ -13,7 +13,7 @@ class RequestHeaderParser
      *
      * @var array
      */
-    private static $allowedMethods = [
+    public static $allowedMethods = [
```
Comment (FaaPz) [inline app/Parsers/RequestHeaderParser/RequestHeaderParser.php]: I don't like this. Why not a simple getter?

(I know, we already have one to return imploded strings...)

**Review by FaaPz — CHANGES_REQUESTED** [review]
Do have a look at my comments. All test cases are fine! We should make another PR with additional test cases to fully test the `RequestHeaderParser` class.

File: .env.travis (hunk 2 above)
Comment (sjongejan) [reply .env.travis:11]: Yes

File: .env.travis (hunk 1 above)
Comment (sjongejan) [reply .env.travis:9]: image

Comment (sjongejan) [reply .env.travis:9]: image

**Review by FaaPz — APPROVED** [review]

**Pull request merged**